request.go

```diff
@@ -247,3 +247,25 @@ type RefundReq struct {
 	Amount    int64  `json:"amount"`
 	Reason    string `json:"reason"`
 }
+
+// Subscribe : Represent Subscribe object payload (request and response)
+type Subscribe struct {
+	ID          string      `json:"id,omitempty"`
+	Name        string      `json:"name"`
+	Amount      string      `json:"amount"`
+	Currency    string      `json:"currency"`
+	CreatedAt   string      `json:"created_at,omitempty"`
+	Status      string      `json:"status,omitempty"`
+	Token       string      `json:"token"`
+	PaymentType PaymentType `json:"payment_type"`
+	Schedule    Schedule    `json:"schedule"`
+}
+
+// Schedule : Represent Schedule object payload
+type Schedule struct {
+	Interval            int    `json:"interval"`
+	IntervalUnit        string `json:"interval_unit"`
+	StartTime           string `json:"start_time,omitempty"`
+	PreviousExecutionAt string `json:"previous_execution_at,omitempty"`
+	NextExecutionAt     string `json:"next_execution_at,omitempty"`
```

Comment:
That last 3 attribute does not seems to be available as request: https://api-docs.midtrans.com/#create-subscription

Reply:
same with answer above, cause it also used as response

Reply:
Let's discuss https://github.com/veritrans/go-midtrans/pull/55#discussion_r361421651